packages/deployer/serverless.aws.yml

```diff
@@ -0,0 +1,39 @@
+service:
+  name: airnode-${opt:providerIdShort}
+
+plugins:
+  - serverless-plugin-typescript
+  - serverless-plugin-optimize
```

Comment:
This both packs the whole thing down to 1/100 of the size and also replaces the monorepo symlinks with actual files.